components/kyma-environment-broker/internal/broker/instance_deprovision.go

```diff
@@ -93,3 +94,10 @@ func (b *DeprovisionEndpoint) Deprovision(ctx context.Context, instanceID string
 		OperationData: operationID,
 	}, nil
 }
+
+func isOngoingDeprovisioning(existingOperation *internal.DeprovisioningOperation) bool {
```

Comment:
if you are checking if it is temporary then the name must be adjusted. The "ongoing" would check only the state

Reply:
👍